install_config/persistent_storage/topics/glusterfs_example_full.adoc

```diff
@@ -0,0 +1,122 @@
+. In your inventory file, set the following variables under `[OSEv3:vars]`:
++
+----
+[OSEv3:vars]
+...
+openshift_registry_selector="role=infra" <1>
+openshift_hosted_registry_storage_kind=glusterfs
+
+openshift_storage_glusterfs_block_deploy=false
+
+openshift_storage_glusterfs_registry_block_deploy=true
+openshift_storage_glusterfs_registry_block_storageclass=true
+openshift_storage_glusterfs_registry_block_storageclass_default=true
+
+openshift_storageclass_default=false
+----
+<1> It is recommended to run the integrated OpenShift Container Registry on
+nodes dedicated to "infrastructure" applications, that is applications deployed
+by administrators to provide services for the {product-title} cluster.
+
+. Add `glusterfs` and `glusterfs_registry` in the `[OSEv3:children]` section to
+enable the `[glusterfs]` and `[glusterfs_registry]` groups:
++
+----
+[OSEv3:children]
+...
+glusterfs
+glusterfs_registry
+----
+
+. Add `[glusterfs]` and `[glusterfs_registry]` sections with entries for each
+storage node that will host the GlusterFS storage. For each node, set
+`glusterfs_devices` to a list of raw block devices that will be completely
+managed as part of a GlusterFS cluster. There must be at least one device
```

Comment:
There appears to be an extra space after "cluster."

Reply:
I don't see it. Did I unintentionally fix it?

Reply:
No, mistake on my part. It looks fine in the build. Thanks.